src/main/kotlin/Shovel.kt

```diff
@@ -48,13 +49,17 @@ fun shovelAsync(consumer: KafkaConsumer<ByteArray, ByteArray>, hbase: HbaseClien
                     }
                     logger.info("Committing offset")
                     consumer.commitSync()
+                    //consumer.resume(consumer.assignment())
                 }
 
                 if (batchCountIsMultipleOfReportFrequency(batchCount++)) {
                     printLogs(consumer, offsets, usedPartitions)
                 }
 
-            } catch (e: HbaseReadException) {
+            } catch (e: HbaseConnectionException) {
+                logger.error("Error connecting to Hbase", e)
+                cancel(CancellationException("Error connecting to Hbase ${e.message}", e))
+            } catch (e: HbaseWriteException) {
                 logger.error("Error writing to Hbase", e)
                 cancel(CancellationException("Error writing to Hbase ${e.message}", e))
```

Comment:
Should we still be cancelling here? Wont this crash the jar still?

Reply:
yes but we are going with the timeout and scale and changes as discussed :-)